### PR TITLE
Add docs/admin/sysctls.md

### PR DESCRIPTION
**Pull request by sttts**
Here is the proposal: https://github.com/kubernetes/kubernetes/pull/26057
Here is the feature: https://github.com/kubernetes/features/issues/34

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1126)

### CODE REVIEW

File: docs/admin/sysctls.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+```
+$ sudo sysctl -a
+```
+
+## Namespaced vs. Node-Level Sysctls
+
+A number of sysctls are _namespaced_ in today's Linux kernels. This means that
+they can be set independently for each pod on a node. Being namespaced is a
+requirement for sysctls to be accessibly in a pod context within Kubernetes.
```
Comment (ingvagabund) [inline docs/admin/sysctls.md]: s/accessibly/accessible

File: docs/admin/sysctls.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+The following sysctls are known to be _namespaced_:
+
+- `kernel.shm*`,
+- `kernel.msg*`,
+- `kernel.sem`,
+- `fs.mqueue.*`,
+- `net.*`.
+
+Sysctls which are not namespaced are called _node-level_ and must be set
+manually by the cluster admin, either by means of the onderlying Linux
```
Comment (ingvagabund) [inline docs/admin/sysctls.md]: s/onderlying/underlying

File: docs/admin/sysctls.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+- `kernel.shm*`,
+- `kernel.msg*`,
+- `kernel.sem`,
+- `fs.mqueue.*`,
+- `net.*`.
+
+Sysctls which are not namespaced are called _node-level_ and must be set
+manually by the cluster admin, either by means of the underlying Linux
+distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DeamonSet
```
Comment (derekwaynecarr) [inline docs/admin/sysctls.md]: s/DeamonSet/DaemonSet

File: docs/admin/sysctls.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+node. This means that setting a _safe_ sysctl for one pod
+
+- must not have any influence on any other pod on the node
+- must not allow to harm the node's health
+- must not allow to gain CPU or memory resources outside of the resource limits
+  of a pod.
+
+By far, most of the _namespaced_ sysctls are not necessarily considered _safe_.
+
+For Kubernetes v1.4, the following set of _safe_ sysctls is supported:
```
Comment (derekwaynecarr) [inline docs/admin/sysctls.md]: s/is/are

**Comment by derekwaynecarr** [comment]
few typos, but lgtm otherwise.

File: docs/admin/sysctls.md (hunk 4 above)
Comment (sttts) [inline docs/admin/sysctls.md]: Sure about this one? The set is singular.

**Comment by derekwaynecarr** [comment]
one last comment, squash if you agree, then LGTM

**Comment by sttts** [comment]
Addressed last comment, squashed and rebased.

**Comment by sttts** [comment]
@kubernetes/docs what's the merge policy here? Anybody can just merge with a lgtm?

**Comment by devin-donnelly** [comment]
I can merge, but if this feature isn't present until the 1.4 release, it should not be merged until then.

I'll do a docs review in the meantime.

**Comment by sttts** [comment]
- removed net.ipv4.tcp_max_syn_backlog to be inline with https://github.com/kubernetes/kubernetes/pull/32072
- rebased

@devin-donnelly will you cherry-pick onto the 1.4 branch? Or should I create another PR for that?

**Comment by sttts** [comment]
Replaced by #1182

**Pull request closed without merging**